temporal-sdk/build.gradle

```diff
@@ -41,21 +41,22 @@ dependencies {
 
     api project(':temporal-serviceclient')
     api group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
-    api group: 'io.micrometer', name: 'micrometer-core', version: '1.6.3'
+    api group: 'io.micrometer', name: 'micrometer-core', version: '1.6.4'
 
     implementation group: 'com.google.guava', name: 'guava', version: '30.1-jre'
+    implementation group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.2'
+    implementation group: 'commons-io', name: 'commons-io', version: '2.8.0'
```

Comment:
Why are we adding these? Are they required by something else?

Reply:
They were part of Google's junit-toolbox. Looks like we don't actually use anything from that toolbox except for Mockito.